google-cloud-datastore/template.mf

```diff
@@ -0,0 +1,13 @@
+Manifest-Version: 1.0
```

Comment:
This OSGi stuff can be weird some times. There are crazy restrictions for the version number IIRC. :-(